test/octet-json.sh

```diff
@@ -4,11 +4,20 @@ tstr="Zenroom test"
 zenroom=$1
 valgrind=$2
 
+qemu_zenroom_run(){
+	qemu-system-arm -M mps2-an385 -kernel src/zenroom.bin -semihosting -nographic -semihosting-config arg="$*"
+	cat ./stdout
```

Comment:
`stdout` might be confusing. You can rename it to `out` or `outlog` (logger of STDOUT).